src/pages/Environments.js

```diff
@@ -666,7 +682,7 @@ export const Environments = ({ nav = undefined }) => {
               }
             },
             {
-              size: { min: '10em', grow: 0 },
+              size: { min: '11em', grow: 0 },
```

Comment:
I suggested this edit while in the file. Prior to this edit, hovering Last Accessed would wrap the d to the next line due to the sort arrow.